cumulusci/cli/cli.py

```diff
@@ -647,6 +655,10 @@ def task_run(config, task_name, org, o, debug):
     if exception:
         raise exception
 
+    if debug_after:
```

Comment:
I like them as independent from each other. Setting debug this way I think would be confusing to the user. Seems reasonable to keep them separate so that you can `debug` at end without worrying about the exceptions. But in that case you'd need to move it before the `raise exception' above.